tools/truncated/package.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "@spectrum-web-components/truncated",
+    "version": "0.41.1",
```

Comment:
While technically correct, the performance test jobs for Chrome and Firefox will continue to fail why this is not `0.0.1`.

Reply:
Will the CI job upgrade it automatically to latest version ? I manually changed it from 0.0.1 to 0.41.1 so it matches the other components 😅

Reply:
Release time tooling ensures that things match when this is merged and release is ready to go.

Reply:
Alrighty, will leave it to 0.0.1